src/pages/Vault/modals/WithdrawFromVestingVaultModal/WithdrawalRequestList/index.tsx

```diff
@@ -0,0 +1,86 @@
+/** @jsxImportSource @emotion/react */
+import React, { useContext } from 'react';
+
+import TEST_IDS from 'constants/testIds';
+import { TokenId } from 'types';
+import Typography from '@mui/material/Typography';
+import { AuthContext } from 'context/AuthContext';
+import { TOKENS } from 'constants/tokens';
+import { useTranslation } from 'translation';
+import { useGetXvsVaultWithdrawalRequests } from 'clients/api';
+import { convertWeiToTokens } from 'utilities';
+import { ConnectWallet, Spinner, LabeledInlineContent } from 'components';
+import { useStyles } from './styles';
+
+export interface WithdrawalRequestListProps {
+  poolIndex: number;
+}
+
+const WithdrawalRequestList: React.FC<WithdrawalRequestListProps> = ({ poolIndex }) => {
```

Comment:
This component doesn't strictly follow our container pattern

Reply:
Indeed!